multicluster/service-mirror/probe_worker.go

```diff
@@ -29,6 +29,8 @@ type ProbeWorker struct {
 
 // NewProbeWorker creates a new probe worker associated with a particular gateway
 func NewProbeWorker(localGatewayName string, spec *multicluster.ProbeSpec, metrics *ProbeMetrics, probekey string) *ProbeWorker {
+	// Mark gateway as enabled
+	metrics.enabled.Set(1)
```

Comment:
Tiny nit: I would rename that metric var to something more expressive, that wouldn't require that comment :grimacing: